katka/serializers.py

```diff
@@ -22,10 +22,43 @@ class TeamSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Team
-        fields = ('public_identifier', 'name', 'group')
+        fields = ('public_identifier', 'slug', 'name', 'group')
 
     def validate_group(self, group):
         if not self.context['request'].user.groups.filter(name=group.name).exists():
             raise PermissionDenied('User is not a member of this group')
 
         return group
+
+
+class EmbeddedTeamSerializer(serializers.Serializer):
+    public_identifier = serializers.UUIDField(required=False)
+    slug = serializers.SlugField(required=False)
```

Comment:
future proofing: we will show both pi and slug, but right now you can only search/select on pi only.